05/pet.py

```diff
@@ -0,0 +1,20 @@
+# PET NAMES
+pet = {'Fluffy': 'hamster', 'Slimy': 'snake', 'Cutie': 'spider'}
+for key, value in pet.items():
+    print('{} is a {}.'.format(key, value))
```

Comment:
V Pythonu3 už jsou používat tzv. f-stringy _(popsané v [PEP-498](https://www.python.org/dev/peps/pep-0498/))_, takže namísto `str.format()` metody jsi mohla použít něco takovéhoto:

```
print(f"{key} is a {value}")
```

... ale to je jen pro zajímavost. Takhle je to také správně. 😉